### ISSUE TITLE
How to cite Experimentology?

### ISSUE THREAD

**Issue by BenStorme** [langcog/experimentology#204]
It's not really an issue, but I didn't find how to cite Experimentology. It would be nice to provide a format for that. For instance, WALS has a very helpful and small rubric on how to cite it, on the main page: see https://wals.info/  Would be nice to have something similar for Experimentology.
Anyway, tanks for this amazing resource!

**Comment by mikabr** [comment]
added semi-manually, doing correctly blocked on https://github.com/quarto-dev/quarto-cli/issues/3838
